File: src/files/FindInitials.java
static void printInitials(String name) {
	if (name.length() == 0)
		return;
	System.out.print(Character.toUpperCase(name.charAt(0)));
	for (int i = 1; i < name.length() - 1; i++)
		if (name.charAt(i) == ' ')
			System.out.print(" " + Character.toUpperCase(name.charAt(i + 1)));
}

public static void main(String args[]) {
	String name = "John Doe";
	printInitials(name);
}

File: src/files/FindInitials.py
def printInitials(name):
	if(len(name) == 0):
		return
	print(name[0].upper()),
	for i in range(1, len(name) - 1):
		if (name[i] == ' '):
			print (name[i + 1].upper()),

def main():
	name = "John Doe"
	printInitials(name)